plugins/services/src/js/components/modals/NewCreateServiceModalServicePicker.js

```diff
@@ -1,144 +1,92 @@
-import jsonServiceImage from '../../../img/service-image-json-medium@2x.png';
-import CosmosPackagesStore from '../../../../../../src/js/stores/CosmosPackagesStore';
-import Image from '../../../../../../src/js/components/Image';
-import Panel from '../../../../../../src/js/components/Panel';
+import CreateServiceModalServicePickerOption from '../../../../../../src/js/components/CreateServiceModalServicePickerOption';
+import CreateServiceModalServicePickerOptionContent from '../../../../../../src/js/components/CreateServiceModalServicePickerOptionContent';
+import CreateServiceModalServicePickerOptionImage from '../../../../../../src/js/components/CreateServiceModalServicePickerOptionImage';
+import defaultServiceImage from '../../../img/icon-service-default-large@2x.png';
+import jsonServiceImage from '../../../img/service-image-json-large@2x.png';
+
+const METHODS_TO_BIND = ['handleServiceSelect'];
 
 class NewCreateServiceModalServicePicker extends React.Component {
   constructor() {
     super(...arguments);
 
-    this.state = {
-    };
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
   }
 
   handleServiceSelect(service) {
     this.props.onServiceSelect(service);
   }
 
-  getCustomServiceGrid() {
-    let customOptions = [{
-      icon: (
-        <Image fallbackSrc={defaultServiceImage} src={defaultServiceImage} />
-      ),
-      label: 'Use the Form',
-      type: 'app'
-    }, {
-      icon: (
-        <Image fallbackSrc={defaultServiceImage} src={defaultServiceImage} />
-      ),
-      label: 'Use the Form to create a MultiContainer',
-      type: 'pod'
-    }, {
-      icon: (
-        <Image fallbackSrc={jsonServiceImage} src={jsonServiceImage} />
-      ),
-      label: 'Enter JSON',
-      type: 'json'
-    }].map((item, index) => {
-      return this.getServiceOption(
-        this.getServiceOptionIconWrapper(item.icon),
-        item.label,
-        index,
-        item
-      );
-    });
-
-    return (
-      <div className="create-service-modal-service-picker-options panel-grid row">
-        {customOptions}
-      </div>
-    );
-  }
-
-  getPackageIcon(cosmosPackage) {
-    return (
-      <Image
-        fallbackSrc={defaultServiceImage}
-        src={cosmosPackage.getIcons()['icon-medium']} />
-    );
-  }
-
-  getServiceDeployOptions() {
-    // TODO: Implement the correct copy when received. DCOS-11807
-    return (
-      <div className="create-service-modal-service-picker container text-align-center">
-        <h4 className="short flush-top">
-          Run your own Service
-        </h4>
-        <p className="lead tall">
-          Create a containerized service or run a command in one of two ways: use our form to be guided through the correct configuration, or enter your JSON configuration directly.
-        </p>
-        {this.getCustomServiceGrid()}
-      </div>
+  render() {
+    const containerClasses = Hooks.applyFilter(
```

Comment:
Can't we adjust this based on the number of registered components? It kinda feels strange to have the packages change this and there is a risk that we have 4 components and a 3x1 layout.